src/components/shared/input/index.tsx

```diff
@@ -47,6 +60,7 @@ const Input = <
   );
   return (
     <View style={styles.containerInput}>
+      <Text>{label}</Text>
```

Comment:
Since label is optional we should check for the prop and then render the text
`{label &&<Text>{label}</Text>)`